blockchain/indexers/manager.go

```diff
@@ -1,874 +0,0 @@
-				parentHash := &block.MsgBlock().Header.PrevBlock
-				parent, err := dbFetchBlockByHash(dbTx, parentHash)
-				if err != nil {
-					return err
-				}
-				cachedBlock = parent
-
-				// When the index requires all of the referenced
-				// txouts they need to be retrieved from the
-				// database.
-				var prevScripts PrevScripter
-				if indexNeedsInputs(indexer) {
-					var err error
-					prevScripts, err = chain.PrevScripts(dbTx, block)
-					if err != nil {
-						return err
-					}
-				}
-
-				// Find out if treasury was enabled at the parent.
-				isTreasuryEnabled, err := chain.IsTreasuryEnabled(parent.Hash())
-				if err != nil {
-					return err
-				}
-
-				// Remove all of the index entries associated
-				// with the block and update the indexer tip.
-				err = dbIndexDisconnectBlock(dbTx, indexer,
-					block, parent, prevScripts,
-					isTreasuryEnabled)
-				if err != nil {
-					return err
-				}
-
-				// Update the tip to the previous block.
-				hash = &block.MsgBlock().Header.PrevBlock
-				height--
-
-				// NOTE: This does not return as it does
-				// elsewhere since it would cause the
-				// database transaction to rollback and
-				// undo all work that has been done.
-				if interruptRequested(ctx) {
-					interrupted = true
-					break
-				}
-			}
-
-			return nil
-		})
-		if err != nil {
-			return err
-		}
-		if interrupted {
-			return errInterruptRequested
-		}
-
-		if initialHeight != height {
-			log.Infof("Removed %d orphaned blocks from %s "+
-				"(heights %d to %d)", initialHeight-height,
-				indexer.Name(), height+1, initialHeight)
-		}
-	}
-
-	// Fetch the current tip heights for each index along with tracking the
-	// lowest one so the catchup code only needs to start at the earliest
-	// block and is able to skip connecting the block for the indexes that
-	// don't need it.
-	bestHeight := int32(chain.BestHeight())
-	lowestHeight := bestHeight
-	indexerHeights := make([]int32, len(m.enabledIndexes))
-	err = m.db.View(func(dbTx database.Tx) error {
-		for i, indexer := range m.enabledIndexes {
-			idxKey := indexer.Key()
-			hash, height, err := dbFetchIndexerTip(dbTx, idxKey)
-			if err != nil {
-				return err
-			}
-
-			log.Debugf("Current %s tip (height %d, hash %v)",
-				indexer.Name(), height, hash)
-			indexerHeights[i] = height
-			if height < lowestHeight {
-				lowestHeight = height
-			}
-		}
-		return nil
-	})
-	if err != nil {
-		return err
-	}
-
-	// Nothing to index if all of the indexes are caught up.
-	if lowestHeight == bestHeight {
-		return nil
-	}
-
-	// Create a progress logger for the indexing process below.
-	progressLogger := progresslog.NewBlockProgressLogger("Indexed", log)
-
-	// At this point, one or more indexes are behind the current best chain
-	// tip and need to be caught up, so log the details and loop through
-	// each block that needs to be indexed.
-	log.Infof("Catching up indexes from height %d to %d", lowestHeight,
-		bestHeight)
```

Comment:
This is still missing the logs that I was referring to.  For example, on `master` it will show up as follows:
```
2021-03-05 05:40:46.659 [INF] INDX: Catching up indexes from height 318967 to 445659
2021-03-05 05:40:56.660 [INF] INDX: Indexed 6899 blocks in the last 10s (117111 transactions, height 325866, 2019-03-11 20:12:20 -0500 CDT)
2021-03-05 05:41:06.660 [INF] INDX: Indexed 7528 blocks in the last 10s (123832 transactions, height 333394, 2019-04-06 16:17:00 -0500 CDT)
...
```